main/main.go

```diff
@@ -35,27 +42,16 @@ func main() {
 	}()
 
 	time.Sleep(1 * time.Second)
```

Comment:
@aarshkshah1992 Why is this sleep needed? Are we trying to wait for the HTTP server to start? I believe the listener `nl` is already accepting new TCP connections by now.

Reply:
@bajtos 

You are right, we can remove this.

Reply:
Cool, I opened a follow-up PR:
https://github.com/filecoin-project/saturn-l2/pull/16

@aarshkshah1992 PTAL